img/media-types.dot

```diff
@@ -1,12 +1,12 @@
 digraph G {
   {
-    manifestList [shape=note, label="Manifest list\n<<optional>>\napplication/vnd.oci.image.manifest.list.v1+json"]
+    imageIndex [shape=note, label="Image Index<<optional>>\napplication/vnd.oci.image.index.v1+json"]
```

Comment:
This dropped a newline with “Manifest list\n” → “Image Index”.  I don't know if that was intentional or not, although my preference would be to keep the newline.